java/README.md

```diff
@@ -11,14 +11,21 @@ This design ensures that updates easily propagate to each language and reduces o
 
 ## System Requirements
 
-The beta release of GLIDE for Valkey was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7 (aarch64-apple-darwin).
+The release of GLIDE for Valkey was tested on the following platforms:
+Linux:
+- Ubuntu 22.04.1 (x86_64)
+- Amazon Linux 2023 (AL2023) (x86_64)
+
+macOS:
+- macOS 12.7 (Apple silicon)
```

Comment:
Should we tell somewhere that "apple silicon" means M1/M2/etc CPUs which mean ARM CPU which mean `aarch_64` arch?

Reply:
Good suggestion, perhaps \something like this - ?

macOS 12.7 (Apple silicon / aarch_64)